models.py

```diff
@@ -25,9 +25,11 @@ def get_todo(db: Session, item_id: int):
     return db.query(ToDo).filter(ToDo.id == item_id).first()
 
 
-def update_todo(db: Session, item_id: int, content: str):
+def update_todo(db: Session, item_id: int, content: str, notes: str = None):
     todo = get_todo(db, item_id)
     todo.content = content
+    if notes is not None:
+        todo.notes = notes
```

Comment:
The `update_todo` function now allows partial updates. Consider adding a check to ensure at least one field is being updated to avoid unnecessary database operations.